### PR TITLE
Tests JLABP

### PR DESCRIPTION
**Pull request by johanlovgren**
Refactored tests for pdfminer/rijndael.py and pdfminer/runlength.py

### CODE REVIEW

File: tests/test_rijndael.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
+import unittest
+from pdfminer.rijndael import RijndaelEncryptor, RijndaelDecryptor
+
+
+class TestArcFourMethods(unittest.TestCase):
```
Comment (gali8033) [inline tests/test_rijndael.py]: Wrong class name probably

File: tests/test_runlength.py [hunk 2]
```diff
@@ -0,0 +1,14 @@
+import unittest
+from pdfminer.runlength import rldecode
+
+
+class TestArcFourMethods(unittest.TestCase):
```
Comment (gali8033) [inline tests/test_runlength.py]: Wrong class name probably

File: tools/latin2ascii.py [hunk 3]
```diff
@@ -12,119 +12,127 @@
...
+    if not args:
+        return usage()
     codec = 'utf-8'
     for (k, v) in opts:
-        if k == '-c': codec = v
+        if k == '-c':
+            codec = v
     for line in fileinput.input(args):
-        line = latin2ascii(unicode(line, codec, 'ignore'))
+        line = latin2ascii(str(line, codec, 'ignore'))
```
Comment (gali8033) [inline tools/latin2ascii.py:132]: good job

File: tests/test_rijndael.py (hunk 1 above)
Comment (johanlovgren) [reply tests/test_rijndael.py]: Wops, copy pasta mistake! Fixed now.

File: tests/test_runlength.py (hunk 2 above)
Comment (johanlovgren) [reply tests/test_runlength.py]: Wops, copy pasta mistake! Fixed now.

**Comment by gali8033** [comment]
Me and Erik approves!

File: tests/test_rijndael.py (hunk 1 above)
Comment (gali8033) [reply tests/test_rijndael.py]: Great!

File: tests/test_runlength.py (hunk 2 above)
Comment (gali8033) [reply tests/test_runlength.py]: Great!

**Review by ErikJungh — APPROVED** [review]

**Review by gali8033 — APPROVED** [review]

**Pull request merged**